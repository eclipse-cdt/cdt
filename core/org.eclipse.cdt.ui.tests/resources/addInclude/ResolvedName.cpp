namespace ns4 {

A a;

}
