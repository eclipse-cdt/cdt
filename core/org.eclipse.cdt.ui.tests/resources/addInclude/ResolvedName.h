namespace ns4 {

class A {};

}
