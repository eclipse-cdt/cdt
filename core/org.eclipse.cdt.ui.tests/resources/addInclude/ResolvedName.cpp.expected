#include "ResolvedName.h"

namespace ns4 {

A a;

}
